.neutrinorc.js

```diff
@@ -4,7 +4,22 @@ const jest = require('@neutrinojs/jest');
 
 module.exports = {
   use: [
-    reactLint(),
+    reactLint({
+      rules: {
+        'no-use-before-define': ["error", { 
+          /**
+           * Ignores reference to functions used before the function declaration. 
+           * Since function declarations are hoisted, this is considered safe.
+           */
+          "functions": false,
+          /**
+           * Ignores a reference if the declaration is in an upper scope, 
+           * but still reports the reference if it is in the same scope as the declaration.
+           */
+          "variables": false 
```

Comment:
This option is configured specifically to enable the arrow functions to be references/used before their declarations. (since the methods within the `SchemaTable` component are declared as:
```js
const renderArray = () => {};
```

Reply:
Let's just allow functions but not variables. We can use `function renderArray() {}` rather than `const renderArray = () => {}`.

Reply:
Applied changes!